### PR TITLE
[tests-only] Used skeleton files wisely on webUISharingInternalUsers feature

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
refactor steps with appropriate skeleton files.
Following suites have been refactored:
- webUISharingInternalUsers

## Related Issue

- Part of owncloud/QA#659

## How Has This Been Tested?

- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13935/

<p>













</p></details>

**Review by Talank — COMMENTED** [review]
The changes LGTM. Also, update the expected failure files.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13954/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13958/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13958/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13983/

<p>





</p></details>

**Comment by kiranparajuli589** [comment]
lgtm 👍🏼

File: tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md [hunk 1]
```diff
@@ -58,31 +58,31 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
--   [webUISharingInternalUsers/shareWithUsers.feature:452](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L452)
--   [webUISharingInternalUsers/shareWithUsers.feature:464](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L464)
--   [webUISharingInternalUsers/shareWithUsers.feature:510](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L510)
--   [webUISharingInternalUsers/shareWithUsers.feature:511](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L511)
--   [webUISharingInternalUsers/shareWithUsers.feature:512](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L512)
+-   [webUISharingInternalUsers/shareWithUsers.feature:136](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L136)
```
Comment (kiranparajuli589) [inline tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md:68]: this test is also failing on oauth2-log oc10server!

File: tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature [hunk 2]
```diff
@@ -289,7 +313,7 @@ Feature: Sharing files and folders with internal users
     When the user clears all error message from the webUI
     And the user shares folder "simple-folder" with user "Brian Murphy" as "Editor" using the webUI
     Then the error message with header 'Error while sharing.' should be displayed on the webUI
-    And user "UserTwo" should not be listed in the collaborators list on the webUI
+    And user "Brian Murphy" should not be listed in the collaborators list on the webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature:316]: good catch, can you also change that in line 312

**Review by individual-it — CHANGES_REQUESTED** [review]
good work, just I noticed a small thing that was not part of your PR but could be easily improved

**Review by individual-it — APPROVED** [review]

**Pull request merged**